src/eventio/scripts/print_simtel_metaparams.py

```diff
@@ -1,17 +1,52 @@
+import sys
+
+import json
 from eventio import EventIOFile
-from argparse import ArgumentParser
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
 from eventio.simtel import HistoryMeta
-from eventio.search_utils import yield_toplevel_of_type
+from difflib import unified_diff
 
-parser = ArgumentParser()
+parser = ArgumentParser(formatter_class=ArgumentDefaultsHelpFormatter)
 parser.add_argument('inputfile')
 parser.add_argument('--encoding', default='utf8', help='Encoding to use for decoding METAPARAMs')
+parser.add_argument('--tel-diff', nargs=2, type=int)
```

Comment:
Technically these two options (`tel-diff` and `json`) are mutually exclusive, but the user has no indication of that. Not a big deal (not many users for this tool), but I mention in case you want to include that.

Reply:
Good point.